### PR TITLE
Enable color in workflows

### PR DESCRIPTION
**Pull request by devpow112**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -66,25 +68,28 @@ jobs:
...
         env:
-          PR_NUMBER: ${{github.event.pull_request.number}}
-          BODY: 'Languages require normalization - pull request #${{steps.changes.outputs.pull-request-number}} has been opened with normalized language files.'
           GITHUB_TOKEN: ${{github.token}}
-  a11y:
+          PR_NUMBER: ${{github.event.pull_request.number}}
+          BODY: >
+            Languages require normalization - pull request
+            #${{steps.changes.outputs.pull-request-number}} has been opened with
+            normalized language files.  a11y:
```
Comment (devpow112) [inline .github/workflows/ci.yml:92]: Ported over some lang normalization clean-up from `d2l-outcomes` as well

File: .github/workflows/update-dependencies.yml [hunk 2]
```diff
@@ -3,9 +3,11 @@ on:
   schedule:
     - cron: '00 10 * * 1-5' # Mon-Fri 6:00am EDT/5:00am EST
   push:
-    branches: master
+    branches: main
```
Comment (devpow112) [inline .github/workflows/update-dependencies.yml:6]: Will revert this

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -66,24 +68,28 @@ jobs:
...
         run: gh pr comment "$PR_NUMBER" -b "$BODY"
         env:
-          PR_NUMBER: ${{github.event.pull_request.number}}
-          BODY: 'Languages require normalization - pull request #${{steps.changes.outputs.pull-request-number}} has been opened with normalized language files.'
           GITHUB_TOKEN: ${{github.token}}
+          PR_NUMBER: ${{github.event.pull_request.number}}
+          BODY: >
+            Languages require normalization - pull request
+            #${{steps.changes.outputs.pull-request-number}} has been opened with
+            normalized language files.
```
Comment (devpow112) [inline .github/workflows/ci.yml:92]: Ported over some lang normalization clean-up from d2l-outcomes as well

**Comment by mpharoah-d2l** [comment]
What does this do? Is this for vdiff?

**Comment by devpow112** [comment]
IT enables colors mainly for the various lint passes. Previously they were just black and white

**Review by mpharoah-d2l — APPROVED** [review]

**Pull request merged**